.spelling

```diff
@@ -519,6 +520,8 @@ intra-cluster
 intrahost
 intranode
 Invasiveness
+iop
+iops
```

Comment:
iop is not a valid word, so these should be removed as well. IOP is an ambiguous abbreviation.

I see the word used in other docs, dos don't necessarily think it was being flagged as an error either.

Reply:
OK. looks like maybe `iops` was flagged. Not sure if IOPs would fair better.